### ISSUE TITLE
Tab Bar icons are not good with some iPhones

### ISSUE THREAD

**Issue by pedroferreira1** [HathorNetwork/hathor-wallet-mobile#221]
User reported that he can't click the icons in the Tab Menu because of the new iOS UI.

![image](https://user-images.githubusercontent.com/3298774/210553926-ca34f754-c06c-4f69-b813-62d521f0099f.png)

iphone 13 pro max
using v0.19.0 of the app

**Comment by glevco** [comment]
I've tested the problem on all iPhone simulators since the iPhone 11 Pro (which is my personal device, that has no problem).

I found that it occurs not only on the iPhone 13 Pro Max, but on all iPhones since the iPhone 12, except for Mini or SE versions.

The iPhone 11 Pro up to the iPhone 12 family:

![Screenshot 2023-01-16 at 20 49 58](https://user-images.githubusercontent.com/1757957/212787118-3fc48b67-67fc-44bb-87e9-940344af19ad.png)

The iPhone 13 and 14 families:

![Screenshot 2023-01-16 at 21 10 21](https://user-images.githubusercontent.com/1757957/212787202-5ff2abd8-c7f6-4f39-9ee5-3be9cbc25b78.png)

So, in total, 10 devices present this problem. Here's a list of their device IDs:

```
'iPhone13,2', // iPhone 12
'iPhone13,3', // iPhone 12 Pro
'iPhone13,4', // iPhone 12 Pro Max
'iPhone14,5', // iPhone 13
'iPhone14,2', // iPhone 13 Pro
'iPhone14,3', // iPhone 13 Pro Max
'iPhone14,7', // iPhone 14
'iPhone14,8', // iPhone 14 Plus
'iPhone15,2', // iPhone 14 Pro
'iPhone15,3', // iPhone 14 Pro Max
```
